src/index.ts

```diff
@@ -30,9 +30,9 @@ export const init = (options: WallyConnectorOptions): void => {
   return;
 };
 
-export const getProvider = (supress?: boolean): WallyConnector | null => {
+export const getProvider = (supress?: boolean): WallyConnector | undefined => {
   if (!checkInjected(supress)) {
-    return null;
+    return undefined;
```

Comment:
just curious, why is undefined better than null?

Reply:
tbh it just felt _cleaner_ haha - I was tryna use `delete` but you can only do that with object properties...and when you `delete` something, it turns into undefined, right? 😅 🤓